source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -448,6 +448,24 @@ def _getFormatFieldAtRange(self, textRange, formatConfig, ignoreMixedValues=Fals
 					if isinstance(textColumnNumber, int):
 						formatField.field['text-column-number'] = textColumnNumber
 		return formatField
+	
+	def _getIndentValueDisplayString(self, val: float) -> str:
+		"""A function returning the string to display in formatting info in Word documents.
+		@param val: an indent value measured in points, fetched via
+			an UIAHandler.UIA_Indentation*AttributeId attribute.
+		@return: The string used in formatting information to report the length of an indentation.
+		"""
+		
+		if self.obj.WinwordApplicationObject:
+			# When Word object model is available we honour Word's options to report distances so that what is
+			# reported by NVDA matches Word's UI (rulers, paragraph formatting dialog, etc.)
+			# Default seem to be inch or centimeters for Western countries localization of Word and characters for
+			# east Asian localisations.
+			return self.obj.getLocalizedMeasurementTextForPointSize(val)
+		
+		# If Word object model is not available, we just fallback to general UIA case, i.e. use Windows regional
+		# settings.
+		super()._getIndentValueDisplayString(val)

```

Comment:
The return was forgotten. After committing this suggestion, please ensure to clearly test the scenario where the object model is unavailable. Even if this is just by temporarily changing the if conndition in this method to `if False:`.
```suggestion
		return super()._getIndentValueDisplayString(val)
```